src/pages/users.tsx

```diff
@@ -173,19 +181,22 @@ function UserEditor(props: {
               {AllRoles
                 .filter(r => isPermitted(me.roles, "PrivilegedRoleManager") || !RoleProfiles[r].privileged)
                 .map(r => {
-                const rp = RoleProfiles[r];
-                return (
-                  <Checkbox key={r} value={r} isChecked={isPermitted(roles, r)} onChange={setRole}>
-                    {rp.privileged ? "*" : ""} {rp.displayName}（{r}）
-                  </Checkbox>
-                );
-              })}
+                  const rp = RoleProfiles[r];
+                  return (
+                    <Checkbox key={r} value={r} isChecked={isPermitted(roles, r)} onChange={setRole}>
+                      {rp.privileged ? "*" : ""} {rp.displayName}（{r}）
+                    </Checkbox>
+                  );
+                })}
             </Stack>
           </FormControl>
         </VStack>
       </ModalBody>
       <ModalFooter>
-        <Button variant='brand' isLoading={saving} onClick={save} isDisabled={!validEmail || !validName}>保存</Button>
+        <Flex justifyContent="space-between" width="100%">
```

Comment:
I think `width="100%"` is not needed?

Reply:
Without this two buttons would be both in the lower right corner